schema.py

```diff
@@ -0,0 +1,118 @@
+
+
+class Account(Base, IdPrimaryKeyMixin, DateTimeMixin):
+    """
+    accounts is the table for the Account objects which hold the user information 
+    """
+    __tablename__='accounts' #Store all account objects in accounts
+
+    # Columns:
+    username = Column(String, nullable = False) #used for personalization
+    password = Column(String, nullable = False) # used for authorization
+    name = Column(String) #So the site can adress you by your chosen name
+    email = Column(String, nullable = False) #Account's email for contacting
+    account_token = Column(String) #Account's Stripe coustomer key
+    admin = Column(Boolean, default = False) #If 0 then this account is not an admin
+
+
+    def __repr__(self):
+        ret = "<ID: %d, username: %s, email: %s, token: %s>" % (self.id, self.username, self.email, self.account_token)
+        return ret
+
+class Student(Base, IdPrimaryKeyMixin, DateTimeMixin):
+    """
+    students is the table for LSES student users information, represented by the Student object
+    """
+    __tablename__ = 'students'
+    
+    # Columns:
+    related_account = Column(Integer, nullable = False) #So we can easily locate the user information
+    open_requests = Column(Integer, default=0) #The number of open requests by the student
+    closed_requests = Column(Integer, default=0) #The number of requests which the student opened that are now closed (filled)
+    gained_money = Column(Float, default=0) #The total amount of money this Student has gained (used for statistics)
+    approved = Column(Boolean, default = False) #There are certain actions an account is denied if unapproved
+
+
+    def __repr__(self):
+        ret = "<ID: %d, Opened Requests: %d, Closed Requests: %d, Money Gained: %d>" % (self.id, self.open_requests, self.closed_requests, self.gained_money)
+        return ret
+
+
+class Donor(Base, IdPrimaryKeyMixin, DateTimeMixin):
+    """
+    donors is the table for people who wish to make donations, represented by Donor objects
+    """
+    __tablename__= 'donors'
+
+    # Columns:
+    related_account = Column(Integer, nullable = False) #So we can easily locate the user information
+    donations_given = Column(Integer, default=0) #The number of requests this donor has closed (been the last one to donate to)
+    money_given = Column(Float, default=0) #The total amount of money this donor has donated
+
+    def __repr__(self):
+        ret = "<ID: %d, donations_given: %d, money_given: %d>" % (self.id, self.donations_given, self.money_given)
+        return ret
+
+class Request(Base, IdPrimaryKeyMixin, DateTimeMixin):
+    """
+    requests is the table which holds all the requests represented by Request objects
+    """
+    __tablename__='requests'
+
+    # Columns:
+    requested_by = Column(Integer) #who gave the request
+    amount_needed = Column(Float) #How much money do you need
+    description = Column(String) #Why the user needs money or requests for physical goods

```

Comment:
I think since we are displaying it on the website it will be easiest to access if we work though this db, since I am trying to connect it up with flask things, but we should discuss with her to see where information is better for the site to grab from.
In today's meeting I would like to discuss this and if so then does the database need the request/donation  functionality or are we best leaving all of that to stripe and then log account information and statistics through the account classes.